packages/configs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@penumbra-zone/configs",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "Shared Penumbra configurations",
+  "repository": "penumbra-zone/web",
+  "type": "module",
+  "scripts": {},
```

Comment:
our current web <> prax local dev workflow is

**web:** `pnpm install && pnpm build && pnpm dev:pack`
**web:** `pnpm dev` 
**prax:** `pnpm -w add:tgz ~/PATH_TO_PRAX_REPO/web/packages/*/penumbra-zone-*.tgz && pnpm -w syncpack fix-mismatches && pnpm dev` 

can we modify the scripts to add build and pack steps so that this package is watched during prax / dex explorer development?

```
  "scripts": {
    "build": "tsc --build --verbose",
    "dev:pack": "tsc-watch --onSuccess \"$npm_execpath pack\"",
  },
```